rudderstack/integrations/destinations/destination_active_campaign_test.go

```diff
@@ -13,28 +13,162 @@ func TestDestinationResourceActiveCampaign(t *testing.T) {
 			TerraformCreate: `
 				api_url = "https://some-url"
 				api_key = "api-key"
+				onetrust_cookie_categories {
```

Comment:
Just to avoid duplicating definitions, these tests usually assume that create defines the required properties, and later update includes optional properties as well. This way we can avoid big payloads in both create and update tests, but test that some properties are marked as optional as well.

Reply:
Addressed.